mxcube3/routes/limsutils.py

```diff
@@ -214,12 +214,8 @@ def get_proposal_info(proposal):
 
 
 def select_proposal(proposal):
-    if not proposal.lower().startswith('mx'):
-	aux_prop = "{}{}".format('mx', proposal)
-    else:
-	aux_prop = proposal
-    
-    proposal_info = get_proposal_info(aux_prop)
+    proposal_info = get_proposal_info(proposal)
```

Comment:
this was not a bug (although it could be improved). It happens that the proposal name can be 'MX12323' or 'mx12323' depending on ispyb (we might have slightly different versions). Please, leave this like it was..

Reply:
Yest but the test is 

`if not proposal.lower().startswith('mx'):`

So it will match all other codes, we are for instance not using mx for all our accounts so we will get
mx prepended to our proposals, we have opid, ix, ifx and so on. So with this code we get mxopid... mxix...

So the prepending of mx needs to be more specific or handled diffrently

Reply:
aahh ok, I see

Reply:
Sorry, it was not exactly for what I said. It is for ensuring that the proposal code is also supplied together with the name. However, I am not sure why that code is there :)
It is not needed from our side, it was earlier probably, so you removal is ok